docs/sources/guides/whats-new-in-v7-0.md

```diff
@@ -12,8 +12,184 @@ weight = -17
+
+Since then we have been working towards removing PhantomJS. In October 2019, when Grafana v6.4 was released, we [announced](https://grafana.com/blog/2019/10/02/grafana-v6.4-released/#phantomjs-deprecation) the deprecation of PhantomJS. Grafana v7.0 removes all PhantomJS support which means that Grafana distribution no longer will include a built-in image renderer.
+
+As a replacement for PhantomJS we’ve developed the [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) which is a plugin that runs on the backend and handles rendering panels and dashboards as PNG images using headless Chromium/Chrome. The [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) can either be installed as a Grafana plugin running in its own process side-by-side with Grafana. or run as an external HTTP service, hosted using Docker or as a standalone application.
+
+Read more about [Image Rendering]({{< relref "../administration/image_rendering/" >}}) in the documentation for further instructions.
+
+## Query History in Explore out of beta
+
+The Query history feature lets you view and interact with the queries that you have previously run in Explore. You can add queries to the Explore query editor, write comments, create and share URL links, star your favorite queries, and much more. Starred queries are displayed in Starred tab, so it is easier to reuse queries that you run often without typing them from scratch.
+
+It was released as a beta feature in Grafana 6.7. The feedback has been really positive and it is now out of beta for the 7.0 release. Learn more about query history in [Explore]({{< relref "../features/explore" >}}).
+
+## Stackdriver data source supports Service Monitoring
+
+[Service monitoring](https://cloud.google.com/service-monitoring) in Google Cloud Platform (GCP) enables you to monitor based on Service Level Objectives (SLOs) for your GCP services. The new SLO query builder in the Stackdriver data source allows you to display SLO data in Grafana. Read more about it in the [Stackdriver data source documentation]({{< relref "../features/datasources/stackdriver/#slo-service-level-objective-queries" >}}).
+
+## Time zone support
+
+Adding support for changing the time zone for a dashboard is one of our oldest feature requests and now in Grafana 7.0, it is finally implemented. You now have the ability to override the time zone used to display date and time values in a dashboard. One of the benefits with this is that you can specify the local time zone of the service/system that you are monitoring which can be really helpful when monitoring a system/service that operates across several time zones.
+
+### Grafana Enterprise
```

Comment:
Why is this an H3? It does not seem like it is part of time zone support. Maybe change to H2 and have the Enterprise feature headings be h3.

Reply:
Totally right - will fix.

Reply:
Done

Reply:
Wait, we only have one Enterprise feature? Then how about

Grafana Enterprise: Usage analytics and presence